### PR TITLE
NMS-14677:Create Velocloud V1 client similar to current V2

### PR DESCRIPTION
**Pull request by aramosvizcarra**
Added client to handle API V1and V2

### CODE REVIEW

File: client/v2/src/main/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientTest.java [hunk 1]
```diff
@@ -25,12 +25,11 @@
  *     http://www.opennms.org/
  *     http://www.opennms.com/
  *******************************************************************************/
-package org.opennms.velocloud;
```
Comment (fooker) [inline client/v2/src/main/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientTest.java:28]: The path of this file is wrong. Should be `src/test/java` not `src/main/test/java`.

File: client/v2/src/main/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientTest.java [hunk 2]
```diff
@@ -42,7 +41,7 @@ public class VelocloudApiClientTest {
```
Comment (fooker) [inline client/v2/src/main/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientTest.java:41]: Class name should be `VelocloudApiClientV2Test`

File: plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java [hunk 3]
```diff
@@ -33,7 +33,8 @@
 
 import org.opennms.integration.api.v1.config.requisition.Requisition;
 import org.opennms.integration.api.v1.config.requisition.immutables.ImmutableRequisition;
-import org.opennms.velocloud.client.VelocloudApiClient;
+import org.opennms.velocloud.client.api.VelocloudApiClient;
+
```
Comment (fooker) [inline plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java:37]: Additional whitespace

**Review by fooker — CHANGES_REQUESTED** [review]

**Comment by amay94** [comment]
So if I understand this correctly, this is just moving the existing client designed for V2 of their API into `client/V2` and creating a V1 version with just the pom and swagger resources?

**Comment by aramosvizcarra** [comment]
> So if I understand this correctly, this is just moving the existing client designed for V2 of their API into `client/V2` and creating a V1 version with just the pom and swagger resources?

Thats right. The resources are the json (v1) and yaml (V2) openapi filde descriptors for each API.
THere is also a new client/api module that will contain all the abstraction that should map to each client so they can be accessed seamlessly by the rest of the application.

**Review by amay94 — APPROVED** [review]
Looks good!

**Review by fooker — APPROVED** [review]

**Pull request merged**